device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x100G/port_config.ini

```diff
@@ -0,0 +1,323 @@
+# name          lanes                           alias           index   speed   fec
+Ethernet0       17                              Ethernet1/1     1       100000  rs
+Ethernet1       18                              Ethernet1/2     1       100000  rs
+Ethernet2       19                              Ethernet1/3     1       100000  rs
+Ethernet3       20                              Ethernet1/4     1       100000  rs
+Ethernet4       21                              Ethernet1/5     1       100000  rs
+Ethernet5       22                              Ethernet1/6     1       100000  rs
+Ethernet6       23                              Ethernet1/7     1       100000  rs
+Ethernet7       24                              Ethernet1/8     1       100000  rs
+Ethernet8       1,2,3,4                         Ethernet2/1     2       400000  rs
```

Comment:
is it possible to skip the non-used ports, so we can have continuous port names?

Reply:
@r12f I will explore options and get back to you.

Reply:
As discussed in yesterday's meeting, we will forego contiguous port names in light of the platform.json complications it raises, marking this as resolved.